geometry/create-and-edit-geometries/src/main/java/com/esri/samples/create_and_edit_geometries/CreateAndEditGeometriesController.java

```diff
@@ -0,0 +1,440 @@
+import com.esri.arcgisruntime.mapping.view.geometryeditor.GeometryEditorTool;
+import com.esri.arcgisruntime.mapping.view.geometryeditor.VertexTool;
+import com.esri.arcgisruntime.symbology.SimpleFillSymbol;
+import com.esri.arcgisruntime.symbology.SimpleLineSymbol;
+import com.esri.arcgisruntime.symbology.SimpleMarkerSymbol;
+
+import javafx.beans.binding.Bindings;
+import javafx.fxml.FXML;
+import javafx.geometry.Point2D;
+import javafx.scene.control.Alert;
+import javafx.scene.control.Button;
+import javafx.scene.control.ComboBox;
+import javafx.scene.input.MouseButton;
+import javafx.scene.paint.Color;
+import javafx.util.StringConverter;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class CreateAndEditGeometriesController {
+
+  @FXML private MapView mapView;
+  @FXML private Button pointButton;
+  @FXML private Button multipointButton;
+  @FXML private Button polylineButton;
+  @FXML private Button polygonButton;
+  @FXML private Button redoButton;
+  @FXML private Button undoButton;
+  @FXML public Button deleteSelectedElementButton;
+  @FXML private Button stopAndSaveButton;
+  @FXML private Button stopAndDiscardButton;
+  @FXML public Button deleteAllGeometriesButton;
+  @FXML private ComboBox<GeometryEditorTool> toolComboBox;
+
+  private GeometryEditor geometryEditor;
+  private VertexTool vertexTool;
+  private FreehandTool freehandTool;
+  private GraphicsOverlay graphicsOverlay;
+  private Graphic selectedGraphic;
+  private SimpleFillSymbol fillSymbol;
+  private SimpleLineSymbol lineSymbol;
+  private SimpleMarkerSymbol pointSymbol;
+  private SimpleMarkerSymbol multipointSymbol;
+
+  public void initialize() {
+
+    // authentication with an API key or named user is required to access basemaps and other location services
+    String yourAPIKey = System.getProperty("apiKey");
+    ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
+    // create a map with the standard imagery basemap style
+    ArcGISMap map = new ArcGISMap(BasemapStyle.ARCGIS_IMAGERY_STANDARD);
+
+    // set the map to the map view
+    mapView.setMap(map);
+
+    // set a viewpoint on the map view
+    mapView.setViewpoint(new Viewpoint(53.08230, -9.5920, 5000));
+
+    // create a graphics overlay for the graphics
+    graphicsOverlay = new GraphicsOverlay();
+
+    // add the graphics overlay to the map view
+    mapView.getGraphicsOverlays().add(graphicsOverlay);
+
+    // create a geometry editor and set it to the map view
+    geometryEditor = new GeometryEditor();
+    mapView.setGeometryEditor(geometryEditor);
+
+    // create vertex and freehand tools for the geometry editor and add to combo box - vertex tool selected by default
+    vertexTool = new VertexTool();
+    freehandTool = new FreehandTool();
+    toolComboBox.setConverter(new ComboBoxStringConverter());
+    toolComboBox.getItems().addAll(vertexTool, freehandTool);
+    toolComboBox.getSelectionModel().select(0);
```

Comment:
```suggestion
    // create vertex and freehand tools for the geometry editor and add to combo box
    vertexTool = new VertexTool();
    freehandTool = new FreehandTool();
    toolComboBox.setConverter(new ComboBoxStringConverter());
    toolComboBox.getItems().addAll(vertexTool, freehandTool);
    // bidirectionally bind the geometry editor tool to the tool selected in the combo box
    toolComboBox.valueProperty().bindBidirectional(geometryEditor.toolProperty());
```
We can move that binding up here now and then removes the need for setting a default. The GeometryEditor is initialized with a VertexTool.